lib/pymedphys/docs/release-notes.md

```diff
@@ -8,6 +8,85 @@ This project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.35.0]
+
+### News around this release
+
+* Installation on MacOS (Intel) has been simplified and is now the same as for other platforms, thanks to work by the new pymssql maintainers.
+
+### "Stable" API changes
+
+(Won't truly be stable until after a 1.0.0 release)
+
+#### Breaking changes
+
+
+#### Deprecations
+
+
+#### New features
+
+* `pymedphys dicom listen` stores incoming DICOM objects in a directory hierarchy: PatientID/Study Instance UID/Series Instance UID
+aligning with the DICOM Q/R hierarchy, and more suitable for use with tools like dicompyler and OnkoDICOM. NOTE: This changes where a 3rd party or in-house program would expect to find the DICOM data on the file system compared to the previous release.
+
+#### Bug fixes
+
+* CLI initialisation was delayed by unused tensorflow imports
+
+### Documenation Changes
+
+#### Logistics changes
+
+* Linked to Discourse group witin tutorials
+* Discourse commenting now available directly within the hosted documentation
+
+### GUI changes
+
+* improved layout and fonts
+* improved demonstrations and tutorials using Jupyter Book
+*  app.pymedphys.com runs live against the main branch of pymedphys
+
+#### Logistics changes
+
+
+
+#### Pseudonymise
+
+##### Bug fixes
+
+* Pseudonymisation of Decimal String (e.g. Patient Weight) was failing
+* Pseudonymisation of Date, Time or DateTime elements with embedded UTC offsets would fail.
```

Comment:
This should probably be scoped under bug fixes within "Experimental API" changes.

Reply:
Ok.
But in the future, can we simplify the release notes to have just
News
Broken public API
New Features/Enhancements
Bug fixes

and then annotation of some sort regarding whether it is
Documentation
Code
Installation

I'm thinking I'd like to start naming/title PRs so that it can be used almost verbatim as the entry in the release notes.

Reply:
Yup, I like that. That heading structure, the annotation, and having PRs named or tagged to make the task easier all makes a lot of sense.

Towncrier is also an interesting approach to this problem which brings us a step closer to having main always be release ready:

https://github.com/twisted/towncrier

Reply:
> But in the future, can we simplify the release notes to have just ...

I wouldn't be opposed to you making those changes here in this release if you'd prefer.